gix-sec/src/lib.rs

```diff
@@ -8,6 +8,7 @@
 #![cfg_attr(all(doc, feature = "document-features"), feature(doc_cfg, doc_auto_cfg))]
 // `unsafe_code` not forbidden because we need to interact with the libc
 #![deny(missing_docs, rust_2018_idioms, unsafe_code)]
+#![cfg_attr(target_os = "wasi", feature(wasi_ext))]
```

Comment:
Why is `wasi_ext` necessary here?

Even though it's probably OK to need nightly for `wasi`, it would be nice if that can be avoided. If not, the existing test can probably be adapted or extended to have nightly-specific tests.

Reply:
The `std::os::wasi::fs` API needs the `wasi_ext`:

<img width="800" alt="image" src="https://github.com/Byron/gitoxide/assets/3468483/37f2a7a5-18d3-4b51-ba50-7ccb030012a3">